admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -1028,7 +1065,26 @@ class AdminModuleController {
     // "Upgrade All" button handler
     $('body').on('click', self.upgradeAllSource, (event) => {
       event.preventDefault();
-      $(self.upgradeAllTargets).click();
+
+      if ($(self.upgradeAllTargets).length <= 0) {
+        console.warn(window.translate_javascripts['Upgrade All Action - One module minimum']);
+        return false;
+      }
+
+      const modulesActions = [];
+      let moduleTechName;
+      $(self.upgradeAllTargets).each(function bulkActionSelector() {
```

Comment:
Looks like this method already exist l.956, a refacto could be done here.

Reply:
Well that's what I initially tried to do, but it's not exactly the same
The clicked elements and HTML structure is a bit different so each case needs to build this array itself